ui/components/app/asset-list-item/asset-list-item.js

```diff
@@ -68,13 +68,17 @@ const AssetListItem = ({
           e.stopPropagation();
           sendTokenEvent();
           dispatch(
-            updateSendToken({
-              address: tokenAddress,
-              decimals: tokenDecimals,
-              symbol: tokenSymbol,
+            updateSendAsset({
+              type: 'TOKEN',
```

Comment:
I think it may be a good idea to make `TOKEN` a constant somewhere; by doing that we don't accidentally use `token` and we can also validate the `type` within the function itself, throwing an error if we don't support that asset type.

Reply:
Oh, I think `ASSET_TYPES` would work